cairo/scripts/deploy_lib.py

```diff
@@ -27,6 +27,10 @@
 EIC_HASH = 0
 NOT_FINAL = 0
 
+admin_address = "0x23a137ff67f2d70c5f87a86f5829af1d2bde86ae9efcedf050fe7a096cd6768"
+admin_pk = "0x3b09a00aab30c4328dbd1b829270e5352dbe450073d2feed81d75ba0b53f4f2"
```

Comment:
both from env?

Reply:
oh true, thanks. added that for reference but forgot to remove them